tests/foreman/api/test_computeresource_libvirt.py

```diff
@@ -156,248 +141,19 @@ def test_positive_create_with_locs(setup):
 
     :CaseLevel: Integration
     """
-    locs = [entities.Location(organization=[setup.org]).create() for _ in range(randint(3, 5))]
-    compresource = entities.LibvirtComputeResource(
-        location=locs, organization=[setup.org], url=setup.current_libvirt_url
-    ).create()
-    assert {loc.name for loc in locs} == {loc.read().name for loc in compresource.location}
-
-
-@pytest.mark.tier2
-def test_positive_create_with_orgs(setup):
-    """Create a compute resource with multiple organizations
-
-    :id: 2f6e5019-6353-477e-a81f-2a551afc7556
-
-    :expectedresults: A compute resource is created with expected multiple
-        organizations assigned
-
-    :CaseImportance: High
-
-    :CaseLevel: Integration
-    """
-    orgs = [entities.Organization().create() for _ in range(randint(3, 5))]
-    compresource = entities.LibvirtComputeResource(
-        organization=orgs, url=setup.current_libvirt_url
+    orgs = [module_target_sat.api.Organization().create() for _ in range(2)]
+    locs = [module_target_sat.api.Location(organization=[orgs[i]]).create() for i in range(2)]
```

Comment:
```suggestion
    locs = [module_target_sat.api.Location(organization=org).create() for org in orgs]
```
Haven't tried locally, but I think this would work also.

Reply:
yeah, it should work